Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm

```diff
@@ -273,7 +286,7 @@ bool isFullWebBrowserOrRunningTest(const String& bundleIdentifier)
     static bool fullWebBrowser = false;
 #endif
 
-    return fullWebBrowser || isRunningTest(bundleIdentifier);
+    return (fullWebBrowser && !isRunningInAppBrowserTest(bundleIdentifier)) || isRunningTest(bundleIdentifier);
```

Comment:
I am a bit confused about this. Who is setting the inappbrowsertest bundle identifier?

Reply:
That is set in the in-app browser tests under http/tests/in-app-browser-privacy

Thanks for reviewing!